scripts/mkcloud

```diff
@@ -808,12 +807,12 @@ function crowbarupgrade_5plus
 {
     if iscloudver 6plus ; then
         onadmin upgrade_prechecks
+        export cloudsource=$upgrade_cloudsource
         onadmin prepare_crowbar_upgrade
         onadmin prepare_cloudupgrade_repos_6_to_7
         onadmin upgrade_admin_backup
         onadmin upgrade_admin_repocheck
         onadmin upgrade_admin_server
-        export cloudsource=$upgrade_cloudsource
```

Comment:
that's probably right. I remember having wrong `cloudsource` values before and that's why I moved it up. It was probably due to not having a complete solution for remove `upgrade_cloudsource`. I'm still trying with this version, but will revert it to the original.